### DISCUSSION TITLE
Program for Laser Ortur Master 2 - Win 10

### DISCUSSION THREAD

**Discussion by DannysJWStore** [arkypita/LaserGRBL#1230] [General]
Hi, i have a problem with instalation program for using Laser Ortur Master 2, I need help :(

**Comment by arkypita** [comment]
Too generic. If you want help please describe your issue.

**Reply by DannysJWStore** [reply]
ok, i got it...its the past . next g : 
How to set up in the Laser Ortur Master 2 - first engraving then cutting?

**Reply by gmmanonymus111** [reply]
I suggest contacting [Ortur Support](https://ortur.tech/support/) if they forget to include a `User Manual` for your engraver.
